tools/bliss-to-ogg-zip.py

```diff
@@ -201,6 +311,23 @@ def main():
     total_duration += duration
     assert rec_filename.startswith(rec_filename_common_prefix) and rec_filename.endswith(rec_filename_common_postfix)
     rec_name = rec_filename[len(rec_filename_common_prefix):-len(rec_filename_common_postfix)]
+    if args.sprint_cache:
+      wav_tmp_filename = "%s/%s/%s_%s.wav" % (dest_dirname, rec_name, seq.start_time, seq.end_time)
+      os.makedirs(os.path.dirname(wav_tmp_filename), exist_ok=True)
+      cmd = ["ffmpeg", "-i", rec_filename, "-ss", str(seq.start_time), "-t", str(duration), wav_tmp_filename,
+             "-loglevel", args.ffmpeg_loglevel]
+      print("$ %s" % " ".join(cmd))
+      check_call(cmd)
+      import soundfile  # pip install pysoundfile
+      audio, sample_rate = soundfile.read(wav_tmp_filename)
+      assert sample_rate == args.raw_sample_rate
+      data = sprint_cache_handler.feature_post_process(numpy.expand_dims(audio, axis=1), seq.segment_name)
+      soundfile.write(wav_tmp_filename, data, args.raw_sample_rate)
+      source_filename = wav_tmp_filename
+      start_time = 0
```

Comment:
I think you should also reset `duration`, or not? Maybe you can set `duration = 'inf'` or so.

Reply:
ffmpeg does not accept inf as a duration. Instead in that case it is possible not to specify a duration at all, which I did now.

However, the duration should be exactly the same as it is the duration of the wav file that is converted, right?

Reply:
I thought setting `duration = 'inf'` would directly work (without the ugly check below). Did you try?
If it does not work, rather than setting it to `"inf"`, `inf` (or `Decimal('Infinity')` in your case, which is more ugly), you should just set it to `None`.

Reply:
There is also nothing like -1 or so? But if there is just no way to do it, then, as I say, just set it to `None`.

Reply:
`duration = 'inf'` does not work as described above. I'll set it to `None` then.